ReleaseProcessAutomation.Tests/Steps/InitialBranching/BranchFromReleaseForContinueVersionStepTests.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.IO;
+using Moq;
+using NUnit.Framework;
+using ReleaseProcessAutomation.Configuration;
+using ReleaseProcessAutomation.Git;
+using ReleaseProcessAutomation.ReadInput;
+using ReleaseProcessAutomation.Scripting;
+using ReleaseProcessAutomation.SemanticVersioning;
+using ReleaseProcessAutomation.Steps.PipelineSteps;
+using Spectre.Console;
+using Spectre.Console.Testing;
+
+namespace ReleaseProcessAutomation.Tests.Steps.InitialBranching;
+
+[TestFixture]
+public class BranchFromReleaseForContinueVersionStepTests
+{
+  [SetUp]
+  public void Setup ()
+  {
+    _gitClientStub = new Mock<IGitClient>();
+    _inputReaderMock = new Mock<IInputReader>();
+    _ancestorStub = new Mock<IAncestorFinder>();
+    _msBuildInvokerMock = new Mock<IMSBuildCallAndCommit>();
+    _continueReleasePatchMock = new Mock<IContinueReleasePatchStep>();
+    _continueReleaseOnMasterStepMock = new Mock<IContinueReleaseOnMasterStep>();
+
+    _consoleStub = new Mock<IAnsiConsole>();
+
+    var path = Path.Join(Environment.CurrentDirectory, c_configFileName);
```

Comment:
Instead of Env.CurrentDir, please use `TestContext.CurrentContext.TestDirectory`. I don't know how often this is done in the tests or if this is the first time. If there's a lot, then please make a separate ticket for this. if it's the first occurance, plaase fix here.

Reply:
Will create a separate ticket